### PR TITLE
Add Ameriprise Financial

### LINKED ISSUE #4077: Add Ameriprise.com
Banking

### PR DESCRIPTION
**Pull request by Rfosu2k2**
Fixes #4077

I did find this while investigating, and they appear to have OTP.  Without having an account I can't verify  if it's sms/email/phone.

https://www.ameriprise.com/privacy-security-fraud/how-we-protect-information/#04

### CODE REVIEW

**Comment by kmpoppe** [comment]
I'll contact them about what "one-time passcodes" means and how you set them up.

File: _data/banking.yml [hunk 1]
```diff
@@ -64,6 +64,13 @@ websites:
       - email
     doc: https://www.americaschristiancu.com/ministry/services/online-banking-bill-pay.html
 
+  - name: Ameriprise Financial
+    url: https://www.ameriprise.com
+    twitter: ameriprise
+    facebook: ameriprise
+    img: ameriprise.png
+    email_address: anti.fraud@ampf.com
```
Comment (Carlgo11) [inline _data/banking.yml]: I don't think the Anti Fraud department is the correct place to contact for this.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
It seems to be SMS 2FA according to https://www.ameriprise.com/privacy-security-fraud/how-we-protect-information/#02
> Each time you log in to the secure site on ameriprise.com, we'll send a verification code, to your enrolled phone number, that you will enter along with your password.

File: _data/banking.yml [hunk 2]
```diff
@@ -58,6 +58,12 @@ websites:
       - email
     doc: https://www.americaschristiancu.com/ministry/services/online-banking-bill-pay.html
 
+  - name: Ameriprise Financial
+    url: https://www.ameriprise.com
+    twitter: ameriprise
+    facebook: ameriprise
```
Comment (Carlgo11) [inline _data/banking.yml]:
```suggestion
    facebook: Ameriprise
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by phallobst** [comment]
Either the branch is corrupted or the _workflow_ check is not correct, @Carlgo11.

Besides, this entry should probably be changed to show 2FA support via SMS (see my [comment](https://github.com/2factorauth/twofactorauth/pull/4279#issuecomment-586486433)).

**Comment by Carlgo11** [comment]
@phallobst author seems gone. Want to make a new PR with the changes you requested?

**Comment by Rfosu2k2** [comment]
> @phallobst author seems gone. Want to make a new PR with the changes you requested?

Still here, just not active much these days.  Feel free to reopen as new, and close this.  Thanks for keeping things progressing "@everyone"

**Comment by phallobst** [comment]
> @phallobst author seems gone. Want to make a new PR with the changes you requested?

Done - see #4960

**Pull request closed without merging**